app-admin/sysrqd/Manifest

```diff
@@ -1 +1,2 @@
-DIST sysrqd-14.tar.gz 4759 BLAKE2B 3ef8b2f5092a899753da487d0a82a71d2d93d0c6a70689b5923fa900f2766de15fa2e416cb667d800dac19499218a0246941b8c52bfe4421af774ccecc62222e SHA512 efad48be3974923ec14d0958e788115c9273bc29b4f03853e6e087f6af872299ea57d42f885be19ff4eeaffa577ecf1c53ea2adb2f84e1487bb2511597b89cb2
+DIST sysrqd-14.tar.gz 4879 BLAKE2B e6653b1bbb4b1f5521d3bb6f525f725378c9f854499ae9574e222164740a5c30c29d089b8fd96f65b50ebc825ce7fbd585af5e457030e734e48648414c98f46c SHA512 9507d44a0b43b999f37d43b52b41b1990d22835dbfb10b4ccc312542f1b4c1f94e62a7f0a141f502e98018561f842252f01c2921eff11eb5a5acc2941ee22b7d
```

Comment:
You can't just change the tarball manifest like that. Instead of going through the trouble, just leave `14` alone and drop it some time after the bump.

Reply:
I see, i've changed the ```SRC_URI``` back to the original link.